iothub/device/src/Authentication/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -64,6 +64,49 @@ internal string ToSignature()
             return BuildSignature(KeyName, Key, Target, TimeToLive);
         }
 
+        internal static string BuildSignature(string audience, string signature, string expiry)
```

Comment:
do we need both this and the method "private string BuildSignature(string keyName, string key, string target, TimeSpan timeToLive)"? They seem to be generating tokens the follow the same format.